verticapy/plotting/_plotly/base.py

```diff
@@ -56,9 +55,28 @@ def _convert_labels_and_get_counts(pivot_array):
         return ids, labels, parents, values
 
     @staticmethod
-    def _get_fig(fig) -> tuple[Figure]:
```

Comment:
why do you erase this function?

Reply:
It was deleted accidentally when merging my two branches.